forms/UrlSearchSelect/index.js

```diff
@@ -86,6 +86,7 @@ export default React.createClass({
 
   getParams(query) {
     return merge(this.props.params, {
+      __jsonp: this.props.jsonp,
```

Comment:
It's used in getJSON. End points requiring aren't always going to use jsonp as an extension. And inferring it from say the presence of a callback parham seems sketchy. I'll add a spec for it